web_scraper/scraper.py

```diff
@@ -0,0 +1,42 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+url = 'https://en.wikipedia.org/wiki/Chernobyl_disaster'
+
+def get_citations_needed_count(url):
+  """
+    A function that will take a URL and scrape through the page to find any "citation needed" clauses within
+  """
+  page = requests.get(url)
+  soup = BeautifulSoup(page.content, 'html.parser')
+  results = soup.find_all(title='Wikipedia:Citation needed')
+  counter = 0
+  
+  for cites_needed in results:
+    counter +=1
```

Comment:
A typical practice when you don't plan on using the variable defined by a for loop is to use an underscore (_) instead, like this:
```Python
for _ in results:
    counter += 1
```
It's a bit weird at first, but it lets the reader know that you don't plan on actually accessing the things you're iterating over.

Also would len() work here instead? Might be slightly faster if so.